.github/workflows/plugins.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   aerospike-node-14:
     runs-on: ubuntu-latest
     container:
-      image: ubuntu:18.04
+      image: ubuntu:22.04
```

Comment:
The versions of aerospike that run on aerospike-node-14 require ubuntu:18.04

Reply:
Ok, then I think the only quick option for now is to use codecov v2 as in the oracle plugin.